src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2973,6 +2973,22 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertifiedPerson_UpdateConsent_Title" = "Ihr Einverständnis";
 
+"HealthCertifiedPerson_UpdateConsent_Delete_Notice" = "Das alte Zertifikat wird automatisch in den Papierkorb geschoben und nach 30 Tagen gelöscht. Das Zertifikat kann innerhalb der 30 Tage jederzeit wiederhergestellt werden.";
+
+"HealthCertifiedPerson_UpdateConsent_Cancel_Notice" = "Falls Sie hier abbrechen, können Sie die Aktualisierung jederzeit in Ihrer Zertifikatsübersicht anfordern.";
+
+"HealthCertifiedPerson_UpdateConsent_Legal_Title" = "Ihr Einverständnis";
+
+"HealthCertifiedPerson_UpdateConsent_Legal_Subtitle" = "Durch Antippen von „Einverstanden“ willigen Sie in folgende Schritte ein:";
+
+"HealthCertifiedPerson_UpdateConsent_Legal_Bullet_1" = "Die App übermittelt das Impfzertifikat zur Überprüfung an das RKI.";
+
+"HealthCertifiedPerson_UpdateConsent_Legal_Bullet_2" = "Das RKI überprüft das Impfzertifikat und stellt ein aktualisiertes Zertifikat aus und übermittelt dieses an Ihre App.";
+
+"HealthCertifiedPerson_UpdateConsent_Bullet_Point_1" = "Die Aktualisierung des Zertifikats ist freiwillig und kostenlos.";
+
+"HealthCertifiedPerson_UpdateConsent_Bullet_Point_2" = "Um sicherzustellen, dass Zertifikate nur einmalig aktualisiert werden können, wird die Kennung des alten Zertifikats durch das RKI dokumentiert.";
+
```

Comment:
Perhaps have a look into #4265, there a moved all texts regarding the reissuance into an own group and renamed it so it matches what @nickguendling mentions: It should be `HealthCertificate_Reissuance_Consent_xyz`

Reply:
Solved